wallet/src/actors/app/methods.rs

```diff
@@ -574,6 +575,26 @@ impl App {
         Ok(())
     }
 
+    /// Handle node status notifications received from a Witnet node.
+    pub fn handle_node_status_notification(&mut self, value: types::Json) -> Result<()> {
+        let status = serde_json::from_value::<types::StateMachine>(value).map_err(node_error)?;
```

Comment:
Feel free to use this syntax if it looks better to you. 
```suggestion
        let status: StateMachine = serde_json::from_value(value).map_err(node_error)?;
```